creator/ingest_runs/models.py

```diff
@@ -77,6 +85,35 @@ class Meta:
         help_text="The user who submitted this ingest run",
     )
 
+    @transition(field=state, source="waiting", target="running")
+    def started(self):
```

Comment:
Is there a reason you chose `started` here? It seems more semantic to use the imperative here as you're telling the job to `start`, which transitions from the `waiting` state to the `running` state. The past participle doesn't seem as indicative to me.

Reply:
I did it to distinguish from the other `start` function that I had there. But I'm not sure if that function was going to be used. I assume it's just old boilerplate code I should delete